tests/test_account.py

```diff
@@ -623,3 +624,71 @@ def test_get_outcome_links_in_context(self, m):
         self.assertIsInstance(outcome_link_list[0], OutcomeLink)
         self.assertEqual(outcome_link_list[0].outcome_group['id'], 2)
         self.assertEqual(outcome_link_list[0].outcome_group['title'], "test outcome")
+
+    # add_grading_standards()
+    def test_add_grading_standards(self, m):
+        register_uris({'account': ['add_grading_standards']}, m)
+
+        title = "Grading Standard 1"
+        grading_scheme = []
+        grading_scheme.append({"name": "A", "value": 90})
+        grading_scheme.append({"name": "B", "value": 80})
+        grading_scheme.append({"name": "C", "value": 70})
+
+        response = self.account.add_grading_standards(title, grading_scheme)
+
+        self.assertIsInstance(response, GradingStandard)
+        self.assertTrue(hasattr(response, 'title'))
+        self.assertEqual(title, response.title)
+        self.assertTrue(hasattr(response, "grading_scheme"))
+        self.assertEqual(response.grading_scheme[0].get('name'), "A")
+        self.assertEqual(response.grading_scheme[0].get('value'), 0.9)
+
+    # add_grading_standards()
```

Comment:
No need to duplicate comments here. All `add_grading_standards` tests should be under a single comment.